docs/data-sources/pubsub_topic.md

```diff
@@ -3,12 +3,12 @@
 page_title: "encore_pubsub_topic Data Source - terraform-provider-encore"
 subcategory: ""
 description: |-
-  Data source that provides information about an Encore-managed resource
+  Encore provisioned pubsub topic information
```

Comment:
We generally write this out as `Pub/Sub`

Reply:
+1